docs/home.html

```diff
@@ -0,0 +1,293 @@
+---
+layout: home.liquid
+title: 'Chainlink Documentation'
```

Comment:
Should we get rid of the top bar with `chainlink documentation` and the feedback bar? Feels duplicative with the header.

Reply:
Removed.